cfme/tests/v2v/test_post_migrations.py

```diff
@@ -57,6 +58,22 @@ def test_migration_post_attribute(appliance, provider, mapping_data_vm_obj_mini,
     source_view = navigate_to(src_vm_obj, "Details")
     summary = source_view.entities.summary("Properties").get_text_of("Container")
     source_cpu, source_socket, source_core, source_memory = re.findall(r"\d+", summary)
+    snap_name = '{}_snap_'.format(src_vm_obj)
+    snap_desc = 'v2v_snap_test'
+    snapshots_before = src_vm_obj.snapshots.count
+
+    src_vm_obj.snapshots.action.create(name=snap_name + '1',
+                                       description=snap_desc, memory=False,)
+    src_vm_obj.snapshots.action.create(name=snap_name + '2',
+                                       description=snap_desc, memory=False,)
+    src_vm_obj.snapshots.action.create(name=snap_name + '3',
```

Comment:
This can be done inside a for loop 
   

         for x in range(0, 2):
             src_vm_obj.snapshots.action.create(name=snap_name + x,
                                       description=snap_desc, memory=False,)